### PR TITLE
Feature/hibernate configuration

### PR DESCRIPTION
**Pull request by czyzu0507**
Hibernate configuration, Pom's  changes and  setting swagger

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,50 +8,122 @@
...
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>${jackson-databind.version}</version>
+        </dependency>
+
+        <!-- https://mvnrepository.com/artifact/postgresql/postgresql -->
+        <dependency>
+            <groupId>postgresql</groupId>
+            <artifactId>postgresql</artifactId>
+            <version>9.1-901-1.jdbc4</version>
```
Comment (danone771909) [inline pom.xml]: in properties

File: robot-db/src/main/java/DBConfiguration/SpringDBConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.orm.hibernate4.HibernateTransactionManager;
+import org.springframework.orm.hibernate4.LocalSessionFactoryBean;
+
+import java.util.Properties;
+
+@Configuration
+@PropertySource("classpath:database.properties")
```
Comment (danone771909) [inline robot-db/src/main/java/DBConfiguration/SpringDBConfiguration.java:17]: :+1:

File: robot-service/pom.xml [hunk 3]
```diff
@@ -7,9 +7,47 @@
...
+    </dependencies>
 
-
+    <build>
+        <plugins>
+            <!-- -agentlib:jdwp=transport=dt_socket,server=y,address=8000,suspend=n-->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>2.3</version>
```
Comment (danone771909) [inline robot-service/pom.xml]: in properties

File: robot-service/src/test/java/TestTest.java [hunk 4]
```diff
@@ -1,11 +0,0 @@
...
-
-import static org.testng.Assert.assertTrue;
-
-public class TestTest {
-
-    @Test
-    public void testTest() {
-        assertTrue(new TestClass().returnTrue());
-    }
-}
```
Comment (danone771909) [inline robot-service/src/test/java/TestTest.java:11]: Overall :+1:

File: pom.xml [hunk 5]
```diff
@@ -8,50 +8,122 @@
...
+        <webapp-runner.version>8.0.33.1</webapp-runner.version>
+        <springframework.version>4.3.2.RELEASE</springframework.version>
+        <servlet-api.version>2.5</servlet-api.version>
+        <springfox-swagger.version>2.2.2</springfox-swagger.version>
+        <javax.servlet-api.version>3.0.1</javax.servlet-api.version>
+        <jackson-databind.version>2.5.0</jackson-databind.version>
+        <json-simple.version>1.1.1</json-simple.version>
+        <common.dbcp.version>1.4</common.dbcp.version>
+        <hibernate.version>5.2.2.Final</hibernate.version>
+        <postgresql.version>9.1-901-1.jdbc4</postgresql.version>
```
Comment (maciejkocur) [inline pom.xml:36]: Nice to have properties :+1:

File: robot-db/pom.xml [hunk 6]
```diff
@@ -2,6 +2,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <packaging>jar</packaging>
     <parent>
         <artifactId>robot</artifactId>
         <groupId>g-shock-team.robot</groupId>
```
Comment (maciejkocur) [inline robot-db/pom.xml:8]: groupId:robot and artifactId:robot, is it in purpose?

**Comment by maciejkocur** [comment]
For me it is perfectly good. :+1:

**Pull request merged**